src/main/java/com/mosaic/ApiResponse.java

```diff
@@ -0,0 +1,63 @@
+package com.mosaic;
+
+import java.time.LocalDateTime;
+
+/**
+ * This is a class for deserializing future json responses from api to java-classes
+ * as api response will be List of forcasts ("intervals") I decided to make forcast itself an inner class of response
+ * example of response is in responseDummy.json under resources/json/.
+ * But im not exactly sure how to make a class which will make sense to deserialize such structure
+ *
+ * Implementation of deserializer with Jackson objectMapper to be done in next step (?)
+ *
+ */
+public class ApiResponse {
+    private WeatherForecast content;
+
+    public ApiResponse () {
+        this.content = new WeatherForecast();
+    }
+
+    public WeatherForecast getContent() {
+        return content;
+    }
+
+
+
+    public class WeatherForecast {
+        private String startTime;
```

Comment:
Minor remark (it is ok to fix it in future changes)
All fields can be final here in order to make instances of this class immutable